api/testdata/complete-scenarios/test5.xml

```diff
@@ -4743,6 +4750,7 @@
                   <Court>
                     <Location>
                       <Place>
+                        <Street>9889 99th Ave. </Street>
```

Comment:
Place where street is added

Reply:
I think everywhere a street got added unexpectedly is a place that has the wrong layout in the test case; needs to be updated to "Offense" to match what the front end would really spit out today.

It's being generated with the "Address" layout and the work-around of #755 is no longer there to prevent the default encoding for "Address".